### DISCUSSION TITLE
Can mosh be patched to fix https://github.com/mobile-shell/mosh/issues/1014

### DISCUSSION THREAD

**Discussion by thegushi** [Homebrew/discussions#2583] [Tap maintenance and Homebrew development]
This seems to bite lots of people, especially on MacOS.  It's a MacOS clock bug.  Mosh hasn't merged the fix yet (which is in https://github.com/mobile-shell/mosh/issues/1158), but has a workaround in 1014.

Here's the problem -- I don't know enough about homebrew to know about how to start suggesting a pull request or anything.  I didn't find anything about mosh in homebrew-patches.  Also, don't know if that's within the "style" of homebrew.  Clue me in please?

**Comment by carlocab** [comment]
Looks like there's a proper patch at https://github.com/mobile-shell/mosh/pull/1124.

@Bo98 would know more about this. It's a bit odd that this bug seemingly affects Catalina but we haven't heard anything about it before.

**Comment by thegushi** [comment]
Okay, so mosh don't want to merge it because it's OS specific.  Apple's not likely to fix their broken clock.  And brew won't fix it because it's not an issue with mainline mosh.

It seems we're at an impasse.

Should I try getting involved in a land war in asia, or going in against a sicilian when death is on the line?

**Reply by SMillerDev** [reply]
> Okay, so mosh don't want to merge it because it's OS specific.

Not sure where you got that. https://github.com/mobile-shell/mosh/pull/1124 looks like it'll be merged soon

**Comment by carlocab** [comment]
Try `brew update && brew upgrade mosh`.

**Comment by thegushi** [comment]
Awesome!  I somehow misread when I thought there was a standoff.  Thank you!
